po/it.po

```diff
@@ -1,297 +1,314 @@
+"+ e Python"
+
+#: pdfarranger/pdfarranger.py:1632
+msgid "Unknown file format"
+msgstr "Formato file sconosciuto"
+
+#: pdfarranger/pdfarranger.py:1644
+msgid "Image files are only supported with img2pdf"
+msgstr "I file di immagine sono supportati solo con img2pdf"
 
-#: pdfarranger/pdfarranger.py:1449
+#: pdfarranger/pdfarranger.py:1656
+msgid "Image format is not supported by img2pdf"
+msgstr "Il formato dell'immagine non è supportato da img2pdf"
+
+#: pdfarranger/pdfarranger.py:1658
+msgid "File is neither pdf nor image"
+msgstr "Il file non è un pdf o una immagine"
+
+#: pdfarranger/pdfarranger.py:1709
 msgid "page"
 msgstr "pagina"
 
 #: data/pdfarranger.ui.h:1
 msgid "Open a file and append it to the current document"
-msgstr "Apri un file e accodalo al documento corrente"
+msgstr "Apri un file e aggiungilo al documento corrente"
 
 #: data/pdfarranger.ui.h:2
 msgid "Save"
 msgstr "Salva"
 
 #: data/pdfarranger.ui.h:3
 msgid "Save As"
-msgstr "Salva Come"
+msgstr "Salva come"
 
 #: data/pdfarranger.ui.h:4
 msgid "Number of Pages"
 msgstr "Numero di Pagine"
 
 #: data/pdfarranger.ui.h:5
 msgid "Main Menu"
-msgstr "Menu Principale"
+msgstr "Menù Principale"
 
 #: data/pdfarranger.ui.h:6
 msgid "Zoom In"
-msgstr "Ingrandisci"
+msgstr "Zoom avanti"
 
 #: data/pdfarranger.ui.h:7
 msgid "Zoom Out"
-msgstr "Rimpicciolisci"
+msgstr "Zoom indietro"
 
 #: data/pdfarranger.ui.h:8
 msgid "Rotate Left"
-msgstr "Ruota a Sinistra"
+msgstr "Ruota a sinistra"
 
 #: data/pdfarranger.ui.h:9
 msgid "Rotate Right"
-msgstr "Ruota a Destra"
+msgstr "Ruota a destra"
 
 #: data/menu.ui.h:1
 msgid "_Import"
 msgstr "_Importa"
 
-#: data/menu.ui.h:2
-msgid "_Save"
-msgstr "_Salva"
-
 #: data/menu.ui.h:3
 msgid "Save _As…"
-msgstr "Salva Come…"
+msgstr "Save come…"
 
 #: data/menu.ui.h:4
 msgid "E_xport Selection…"
-msgstr "E_sporta Selezione…"
+msgstr "Esporta Selezione…"
 
 #: data/menu.ui.h:5
 msgid "_Undo"
-msgstr "_Annulla"
+msgstr "Annulla"
 
 #: data/menu.ui.h:6
 msgid "_Redo"
-msgstr "_Rifai"
+msgstr "_Ripristina"
 
 #: data/menu.ui.h:7
 msgid "_Edit"
-msgstr "_Modifica"
+msgstr "Modifica"
 
 #: data/menu.ui.h:8
 msgid "Rotate _Left"
-msgstr "Ruota a _sinistra"
+msgstr "Ruota a sinistra"
 
 #: data/menu.ui.h:9
 msgid "_Rotate Right"
-msgstr "_Ruota a destra"
+msgstr "Ruota a destra"
 
 #: data/menu.ui.h:10
 msgid "Cro_p"
 msgstr "Ritaglia"
 
 #: data/menu.ui.h:11
 msgid "_Delete"
-msgstr "_Cancella"
+msgstr "Cancella"
 
 #: data/menu.ui.h:12
 msgid "Dupl_icate"
-msgstr "Duplicato"
+msgstr "Duplica"
 
 #: data/menu.ui.h:13
 msgid "Re_verse Order"
 msgstr "Inverti Ordine"
 
 #: data/menu.ui.h:14
 msgid "_Split pages"
-msgstr "_Dividi pagine"
+msgstr "Dividi Pagine"
 
 #: data/menu.ui.h:15
 msgid "Cu_t"
-msgstr "Taglia"
+msgstr "_Taglia"
 
 #: data/menu.ui.h:16
 msgid "_Copy"
 msgstr "_Copia"
 
 #: data/menu.ui.h:17
 msgid "Paste _After"
-msgstr "Incolla_Dopo"
+msgstr "Incolla prima"
```

Comment:
Did you switch prima/dopo on purpose? Just using Google translate, I think after = dopo; before = prima was correct, but I trust you if you tell me it was done on purpose.

Reply:
This is not "outdated" it's just more obvious in the first commit.